src/api/ingest/utils.py

```diff
@@ -75,6 +76,9 @@ def import_table(datafile, identifier, cleardata, tofile):
             print("There was an error in importing row {}. Import failed. See details below.".format(rowcount))
             print(e.args[0])
             sys.exit()
+        except ValueError as e:
```

Comment:
You should be able to do, on line 75:
`except IntegrityError, ValueError as e:`

Reply:
Ok!

Reply:
Hold on, I wanted to give different messages for these but maybe I can just leave it with the generic message